compiler/build.gradle

```diff
@@ -62,6 +62,9 @@ dependencies {
     implementation(project(":ts-model"))
     implementation(project(":idl-lowerings"))
     implementation(project(":ts-lowerings"))
+    implementation(project(":js-parser"))
+    implementation(project(":js-declarations"))
+    implementation(project(":js-lowerings"))
```

Comment:
Not that important - for instance Alexey is not enthusiastic about this as well - and as you can see this rule is quite often not respected but please tend to insert new list entities in a lexicographic order. 

Any other reader will spend some milliseconds lesser to finding what he's looking for if needed.